sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -427,4 +428,14 @@ public void clearWebElement(By selector) {
       }
     }
   }
+
+  public String getTextFromLabelIfCheckboxIsChecked(By checkbox) {
+    waitUntilIdentifiedElementIsPresent(checkbox);
```

Comment:
Please delete this line because `waitUntilIdentifiedElementIsPresent` is already called inside of `scrollToElement` method.

Reply:
done.